src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -305,6 +305,23 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
               )}
             </Box>
           )}
+          {reserve.symbol == 'stETH' && (
+            <Box>
+              <Warning severity="info">
+                <AlertTitle>Staking Rewards</AlertTitle>
```

Comment:
I think this may also need to be wrapped in `<Trans>` tags as well.

Reply:
Added in trans. 

Would need help with the warning component